### ISSUE TITLE
Multiple paths warning when launching, even though only one has been specified.

### ISSUE THREAD

**Issue by nicolassatragno** [jsimone/webapp-runner#72]
Running

```
java -Dgrails.env=prod -jar build/server/webapp-runner-*.jar --expand-war --port 8000 libs/war_file.war
```

Yields

```
WARNING: multiple paths are specified, but no longer supported. First path will be used.
```

And fails to load the WAR file in the container. This seems to have been broken recently as it works in `8.5.5.2` but is broken in `8.5.11.1`.

**Comment by jkutner** [comment]
@nicolassatragno Can you check if the path `build/server/webapp-runner-*.jar` matches multiple files? If it does, I think it will be expanded, which might result in the error you described.

**Comment by jkutner** [comment]
@nicolassatragno also, can you share your `build.gradle`? If there are multiple files in `build/server` I want to fix that.

**Comment by jkutner** [comment]
@nicolassatragno ok, i was able to reproduce this. As a workaround, if you change your `build.gradle` to include the following it should work:

```
task copyToLib(type: Copy) {
    into "$buildDir/server"
    from(configurations.compile) {
        include "webapp-runner-8*"
    }
}
```

But i'll fix the project's `pom.xml` so the other JARs don't get copied in by default.

**Comment by nicolassatragno** [comment]
> Can you check if the path build/server/webapp-runner-*.jar matches multiple files?

Ah! That was it

>  if you change your build.gradle to include the following it should work:

Right, making sure it only copies the `webapp-runner-<version>` will fix it for now. Thanks a bunch!

**Comment by jkutner** [comment]
Fixed by a1eed0e5059c09b0296651198e48f4b7aa07d31b and released in 8.5.11.2

@nicolassatragno can you give 8.5.11.2 and let me know if it works or not? thanks.

**Comment by nicolassatragno** [comment]
Confirmed the new version works, thanks!
